### DISCUSSION TITLE
API Access?

### DISCUSSION THREAD

**Discussion by MundoMedia** [FreshRSS/FreshRSS#6677] [Q&A]
Hello,

I was building a python script to access my new Fresh RSS installation, and thought I would try using the API.
I opened settings and went to Authentication and 'ticked' the 'Allow Access to API'
Then visited the 'Profile > API Management' section and visited the API link which displays a 'Pass' for both Google Reader and Fever, so I figured something is working.

I wasn't clear from the documentation what needs to be done re the password.
There is 'API Passowrd' then an empty field and a small 'key' icon button to the right -- whcih seems to show the password when you enter one.
So I entered a new password then 'submit' at the bottom (to save it?) and the page comes back with the password field empty again, and no acknowledgement that a password has been entered or accepted.

But, anyway, I tried accessing it with my script but keep getting an authentication error -- so, am I missing something here?
Thanks.

**Comment by Alkarex** [comment]
Passwords are not re-shown to users after they are saved.
Regarding your script, try first to curl manually https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html#google-reader-compatible-api
